### PR TITLE
Refine card

### PR DESCRIPTION
**Pull request by henryzbw**
image

### CODE REVIEW

File: src/components/TimeCorner/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import { withStyles } from '@material-ui/core';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import styles from './styles';
+import { getEndTimeCornerString } from '../../helpers';
+
+@injectIntl
+@withStyles(styles, { withTheme: true })
+@inject('store')
+@observer
+export default class TimeCorner extends Component {
```
Comment (dwalintukan) [inline src/components/TimeCorner/index.js]: nit: got a better name for this? Can this just be called TimeLeft?

File: src/helpers/utility.js [hunk 2]
```diff
@@ -153,6 +153,26 @@ export function getEndTimeCountDownString(unixDiff, locale, localeMessages, isSh
   return moment.duration(unixDiff, 'seconds').format(`d[${formatMessage(day)}] h[${formatMessage(hour)}] m[${formatMessage(minute)}] s[${formatMessage(second)}]`);
 }
 
+/**
+ * Converts a timestamp to the display string.
+ * @param unix {Number} The timestamp to convert. Formatted in unix time format.
+ * @param locale {Object}} Locale object that the Intl is using.
+ * @param localeMessages {Object} LocalMessages object that the Intl is using.
+ * @param isShort {Boolean} Whether to show shorter version.
```
Comment (dwalintukan) [inline src/helpers/utility.js]: delete unused `@params`

File: src/helpers/utility.js [hunk 3]
```diff
@@ -153,6 +153,26 @@ export function getEndTimeCountDownString(unixDiff, locale, localeMessages, isSh
...
+ * @return {Array} An array inclue time and date
```
Comment (dwalintukan) [inline src/helpers/utility.js]: nit: obj would better structure. { time, date }

File: src/models/MultipleResultsEvent.js [hunk 4]
```diff
@@ -74,4 +75,16 @@ export default class MultipleResultsEvent {
       default: throw Error(`Invalid status: ${this.status}`);
     }
   }
+
+  getEventDesc = () => {
```
Comment (dwalintukan) [inline src/models/MultipleResultsEvent.js]: this seems more suited to be in the Card Component?

File: src/components/EventCard/styles.js [hunk 5]
```diff
@@ -65,12 +65,16 @@ const styles = (theme) => ({
     marginRight: theme.padding.spaceX.px,
   },
   eventCardInfo: {
-    position: 'absolute',
-    bottom: theme.padding.space3X.px,
+    bottom: theme.padding.space2X.px,
     color: theme.palette.text.primary,
+    position: 'absolute',
+  },
+  alignBottom: {
```
[the remark this answers is not in the document]
Comment (henryzbw) [reply src/components/EventCard/styles.js]: will do that later. need to merge first in order to continue with other issue.

File: src/components/Progress/styles.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+
+
+const styles = (theme) => ({
+  invalid: {
+    '& > div': {
```
[the remark this answers is not in the document]
Comment (henryzbw) [reply src/components/Progress/styles.js]: i just copied it from other place, not changed it at all.

File: src/components/TimeCorner/index.js (hunk 1 above)
Comment (henryzbw) [reply src/components/TimeCorner/index.js]: its the time itself, not time left, so i think its the time at the corner, so timecorner make sense.

**Pull request merged**